app/models/user.rb

```diff
@@ -9,6 +9,6 @@ class User < ApplicationRecord
   has_one :cart
   has_one :order
   has_one :wishlist
-  has_many :addresses
+  has_many :useraddresses
   has_many :products
 end
```

Comment:
please use one blank line after the end of the file

Reply:
okk